nautobot/ipam/tests/test_filters.py

```diff
@@ -258,6 +258,11 @@ def test_vrf(self):
     def test_present_in_vrf(self):
         # clear out all the randomly generated route targets and vrfs before running this custom test
         test_prefixes = list(self.queryset.values_list("pk", flat=True)[:10])
+        # With advent of `Prefix.parent`, Prefixes can't just be bulk deleted without clearing their
+        # `parent` first in an `update()` query which doesn't call `save()` or `fire `(pre|post)_save` signals.
+        unwanted_prefixes = self.queryset.exclude(pk__in=test_prefixes)
+        unwanted_prefixes.update(parent=None)
+        unwanted_prefixes.delete()
         self.queryset.exclude(pk__in=test_prefixes).delete()
```

Comment:
Redundant after the above addition?

Reply:
Yes, thanks. You're catching all my self-review items! 💋